modin/error_message.py

```diff
@@ -33,17 +33,20 @@ def not_implemented(cls, message=""):
         )
 
     @classmethod
-    @logger_decorator("MODIN-ERROR", "ErrorMessage.single_warning", "debug")
     def single_warning(cls, message):
         message_hash = hash(message)
+        logger = get_logger()
+        logger.debug(f"Modin Warning: Single Warning: {message} was raised.")
```

Comment:
This line should be moved below the return - my thoughts were that this is logged if the warning is raised, while the `...suppressed` one is raised if it is suppressed (so an either or situation).